src/phoenix/experimental/evals/functions/classify.py

```diff
@@ -166,13 +170,20 @@ async def consumer(
                     continue
                 else:
                     tqdm.write("Worker timeout, requeuing")
-                    await queue.put(item)
-            except Exception:
-                tqdm.write(f"Exception in worker: {traceback.format_exc()}")
-                if self.exit_on_error:
-                    self._TERMINATE.set()
+                    await queue.put((self.requeue_priority, item))
```

Comment:
```suggestion
                    await queue.put((priority - 1, item))
```

`self.requeue_priority` seems unnecessary.

Reply:
I'm going to leave task timeout requeues as uncounted to preserve current behavior, let's revisit when we remove `tenacity`